packages/expo/bin/autolinking

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env node
 
+// Executes expo-modules-autolinking CLI directly
+// WARN: This is used in expo-templates-bare-minimum (prebuild) templates and expo-modules-autolinking's expo-gradle-plugin
```

Comment:
```suggestion
```

that is not quite true. we also manually invoke `npx expo-modules-autolinking` from command line to verify the autolinking result. it calls into this `expo/bin/autolinking`. so that's fine to not having the comment here.